test/methods/geom_relations.jl

```diff
@@ -156,14 +156,16 @@ test_pairs = [
 
 function test_geom_relation(GO_f, LG_f, f_name; swap_points = false)
     for (g1, g2, sg1, sg2, sdesc) in test_pairs
-        if swap_points
-            g1, g2 = g2, g1
-            sg1, sg2 = sg2, sg1
+        @test_all_implementations (g1, g2) begin
+            if swap_points
+                g1, g2 = g2, g1
+                sg1, sg2 = sg2, sg1
+            end
+            go_val = GO_f(g1, g2)
+            lg_val = LG_f(g1, g2)
+            @test go_val == lg_val
+            go_val != lg_val && println("\n↑ TEST INFO: $sg1 $f_name $sg2 - $sdesc \n\n")
         end
```

Comment:
This is killing test logs - it is literally impossible to read them because of the number of testsets.  There must be a better way to handle this case - maybe using an external loop over the modules instead of an internal one over the geometries?

Reply:
Inverting the loop wont change the amount of printing as its in the middle already.

I think we instead need to remove that line? Maybe we could make it a testset instead?

Reply:
I don't think the printing is an issue because as we fix the test cases, those will naturally fix themselves.  I guess the problem is that each case has an individual testset - we should change that so that each module just runs all the test cases instead.

Maybe in this case we could use an explicit loop instead of the macro?